packages/components/psammead-media-player/src/Guidance/index.jsx

```diff
@@ -61,15 +61,15 @@ const GuidanceMessage = styled.span`
 
 const IconWrapper = styled.div``;
 
-const Guidance = ({ message, service, children }) => {
+const Guidance = ({ guidanceMessage, service, children }) => {
   return (
     <StyledGuidance service={service}>
```

Comment:
Zeplin says the font size should be equal to 15px. It's showing as 12px for me:

![Screenshot 2019-10-31 at 12 26 45](https://user-images.githubusercontent.com/1430787/67946673-baaebe80-fbd9-11e9-8dbb-f1e1998573b7.png)

If you're not sure which size to import from `@bbc/gel-foundations/typography`, please speak to a UX Designer.